### ISSUE TITLE
Feature request: Parse social links from ENS as well as other data

### ISSUE THREAD

**Issue by philipmather** [frenspage/frenspage#6]
As a User who has an ENS domain I would like my social buttons and links to be parsed automagically from my txt records and presented immediately as defaults to the "want to add your twitter acc, ser?" flow and all similar future flows.

Hence upon setting an ENS domain identity it would resolve the name for all text records, parsing any that either...
1) Match a whitelist.
or
2) Conform to the format ${validWeb2TLD}.${validDomain}, e.g. "com.twitter", "com.github", "org.telegram" etc...

Hence producing suitable links for those that then resolve with a HTTP 200 call to "https://${validDomain}.${validTLD}/{txtRecordValue} or by hardcoded transformation to other "well known" vendor paths e.g. "https://www.linkedin.com/in/philipmather/" or "https://www.reddit.com/user/philipmather"

Other considerations...
1) Use the favicon or hardcoded logo for web 2 links.
2) Use the "avatar" record for web 3 links.
3) Use the other ENS Standard values for other default responses as per https://docs.ens.domains/contract-api-reference/publicresolver#get-text-data, the Global and Service keys as per https://eips.ethereum.org/EIPS/eip-634
4) Very careful input santization of txt records to avoid DNS/ENS applification attacks.
5) The guidelines as described specifically in https://docs.ens.domains/dapp-developer-guide/front-end-design-guidelines#what-to-do-if-the-reverse-record-doesnt-correspond-to-the-forward-resolution etc...

**Comment by mitchoz** [comment]
Thx a lot for this great input!
We had actually already planned more ENS integrations and will soon release a new mirror article explaining our thought process.

TLDR; today I feel like we should not yet integrate all of the ENS data/features right away because it can make the profiles very crowded and distract from their actual purpose (especially during the upcoming phase 2).
